filter/lua.go

```diff
@@ -0,0 +1,353 @@
+package filter
+
+import (
+	"fmt"
+	"runtime"
+	"sync"
+	"sync/atomic"
+
+	"github.com/AdRoll/baker"
+
+	lua "github.com/yuin/gopher-lua"
+)
+
+var luaHelp = `
+This filter runs a baker filter defined in a LUA script.
+It's useful to quickly write and run a Baker filter without having to recompile Baker.
+
+This filter is based on [GopherLua](github.com/yuin/gopher-lua), which is an LUA5.1 virtual machine.
+
+To use this filter you need to declare a function in an lua file. This function serves the same purpose
```

Comment:
maybe it's worth telling the user that the function name must be the same as `FilterName` conf

Reply:
I'm not sure I see how can the user could be confused here since the config h as 2 fields: a filename and a function name.